lib-ui-compose/build.gradle.kts

```diff
@@ -122,6 +122,9 @@ android {
         jvmToolchain(17)
     }
 }
+//dependencies {
```

Comment:
Please remove this. Generally we should not have commented out code in commits. If we need to refer to what the code was at some point before, then that is what Git is for. The only exception is when there is specific code that we would restore once some known issue is resolved.
Whenever we have commented out code, there should be a comment before it explaining why it is commented out instead of just removed.